commerce_coordinator/apps/commercetools/views.py

```diff
@@ -52,11 +52,17 @@ def post(self, request):
 
         try:
             order = client.get_order_by_id(order_id)
-        except CommercetoolsError as _:  # pragma no cover
-            logger.error("Order not found: %s", order_id)
+        except CommercetoolsError as err:  # pragma no cover
+            logger.error("[CT-OrderFulfillView] Order not found: %s", order_id)
+            logger.error(f'[CT-OrderFulfillView] CT error {err}, {err.errors}')
             return Response(status=status.HTTP_404_NOT_FOUND)
 
-        customer = client.get_customer_by_id(order.customer_id)
+        try:
+            customer = client.get_customer_by_id(order.customer_id)
+        except CommercetoolsError as err:  # pragma no cover
+            logger.error("[CT-OrderFulfillView] Customer not found: %s for order %s", order.customer_idm, order_id)
```

Comment:
@grmartin nit: `order.customer_idm`

Reply:
i cought this in the lower case but not here... ill mirror and get a new one up... ty!